packages/sitecore-jss/src/personalize/utils.ts

```diff
@@ -55,27 +55,46 @@ export class CdpHelper {
    * @param {string} pageId the page id
    * @param {string} language the language
    * @param {string} variantId the variant id
+   * @param {string} [scope] the scope value
    * @returns {string} the formatted page variant id
    */
-  static getPageVariantId(pageId: string, language: string, variantId: string): string {
+  static getPageVariantId(
+    pageId: string,
+    language: string,
+    variantId: string,
+    scope?: string
+  ): string {
     const formattedPageId = pageId.replace(/[{}-]/g, '');
     const formattedLanguage = language.replace('-', '_');
+    const scopeId = scope ? `${this.getScope(scope)}_` : '';
```

Comment:
Really more of a nitpick, but would it be better to add the underscore in getScope function?

Reply:
I expect, potentially, that customer can reuse _getScope_ in a sample app, so decided just to make it more common

Reply:
@illiakovalenko makes sense then. Since customers can get scope from env too, maybe getScope can instead be called something like "normalizeScope" or "transformScope" since it ensures scope string doesn't have restricted chars?
Just a thought, I think we can go with the current implementation as it is too.

Reply:
@art-alexeyenko I like that! Renamed to _normalizeScope_, I was struggling on how to call it